internal/pkg/describe/status.go

```diff
@@ -385,27 +395,40 @@ func (s *ecsServiceStatus) writeTaskSummary(writer io.Writer) {
 }
 
 func (s *ecsServiceStatus) writeRunningTasksSummary(writer io.Writer, primaryDeployment awsecs.Deployment, activeDeployments []awsecs.Deployment) {
-	header := "Running"
-	var (
-		barData            []int
-		barRepresentations []string
-	)
+	var data []summarybar.Datum
 	if len(activeDeployments) > 0 {
 		var runningPrimary, runningActive int
 		for _, d := range activeDeployments {
 			runningActive += (int)(d.RunningCount)
 		}
-
 		runningPrimary = (int)(primaryDeployment.RunningCount)
-		barData = []int{runningPrimary, runningActive}
-		barRepresentations = []string{color.Green.Sprint("█"), color.Blue.Sprint("█")}
+		data = []summarybar.Datum{
+			{
+				Value:          runningPrimary,
+				Representation: color.Green.Sprint("█"),
+			},
+			{
+				Value:          runningActive,
+				Representation: color.Blue.Sprint("█"),
+			},
+		}
 	} else {
-		barData = []int{(int)(s.Service.RunningCount), (int)(s.Service.DesiredCount) - (int)(s.Service.RunningCount)}
-		barRepresentations = []string{color.Green.Sprint("█"), color.Green.Sprint("░")}
+		data = []summarybar.Datum{
```

Comment:
can we remove this else-statement? like set this by default at l.398 and then replace it when the if-statement evaluates to true